cli/bench/insert.go

```diff
@@ -0,0 +1,20 @@
+package bench
+
+import (
+	"sync"
+
+	"github.com/FZambia/tarantool"
+	"github.com/tarantool/cartridge-cli/cli/common"
+	"github.com/tarantool/cartridge-cli/cli/context"
+)
+
+func InsertRequestsLoop(ctx context.BenchCtx, tarantoolConnection *tarantool.Connection, results *Results, waitGroup *sync.WaitGroup, benchActiveFlag *bool) {
```

Comment:
it seems to me that the `becnhActiveFlag` check should be before the function call.